Add documentation for online volume expansion (#826)
harvester/docs: docs/rancher/csi-driver.md

:::

> ## Volume Online Reszing ##
> 
> ### Prerequisites
> 
> - Harvester v1.6 or later must be installed on the host cluster.
> - Harvester CSI driver v0.1.24 or later is required.
> 
> If the underlying host storage provider supports [online volume expansion](../volume/edit-volume.md#volume-online-expansion), you can expand an `RWO` (ReadWriteOnce) volume in the guest cluster even while it is attached to a running workload.

Reviewer: ```suggestion
## Volume Online Reszing ##

## Online Volume Resizing

If the underlying storage provider supports [online volume expansion](../volume/edit-volume.md#online-volume-expansion), you can expand a ReadWriteOnce (RWO) volume in the guest cluster even while it is attached to a running workload.

### Prerequisites

- Harvester v1.6 or later
- Harvester CSI Driver v0.1.24 or later
```
